ios-showcase-template/Info.plist

```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>$(DEVELOPMENT_LANGUAGE)</string>
 	<key>CFBundleDisplayName</key>
-	<string>AeroGear&#x2007;iOS&#x2007;Showcase</string>
+	<string>AeroGeariOSShowcase</string>
```

Comment:
I think we can use space here. It just won't display on the home screen, but other places should be ok